packages/mobile/src/onboarding/registration/EnableBiometry.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { NavigationContainer } from '@react-navigation/native'
+import { fireEvent, render } from '@testing-library/react-native'
+import * as React from 'react'
+import { BIOMETRY_TYPE } from 'react-native-keychain'
+import { Provider } from 'react-redux'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import EnableBiometry from 'src/onboarding/registration/EnableBiometry'
+import { createMockStore, getMockStackScreenProps } from 'test/utils'
+
+const mockScreenProps = getMockStackScreenProps(Screens.EnableBiometry)
+
+const renderComponent = () => {
+  const store = createMockStore({
+    app: {
+      supportedBiometryType: BIOMETRY_TYPE.FACE_ID,
+      biometryEnabled: true,
+      activeScreen: Screens.EnableBiometry,
+    },
+    account: {
+      choseToRestoreAccount: false,
+    },
+  })
+
+  return render(
+    <Provider store={store}>
+      <NavigationContainer>
```

Comment:
I wanted to add a test for the skip button in the screen header but i cannot get it to render even with this here - any ideas? when i debug what is rendered by the `render` it starts from the ScrollView of the screen, not with navigation wrapper....

Reply:
This is because the screen here is not being rendered via `react-navigation`.
We would need to create a stack for testing with the route.

Something like:
```tsx
<NavigationContainer>
      <Stack.Navigator>
        <Stack.Screen
          name={Screens.EnableBiometry}
          component={ EnableBiometry }
          initialParams={ params }
        />
      </Stack.Navigator>
    </NavigationContainer>
```

But there might be some additional problem with this.

Reply:
great idea!! we actually have a mocked navigator (unused) which i could refurbish and use here, works like a charm :D am glad that i could cover the skip button, since it's unlikely this can be covered by e2e